versions/3.1.0.md

```diff
@@ -2241,10 +2243,18 @@ description: Pets operations
 
 A simple object to allow referencing other components in the OpenAPI document, internally and externally.
 
+The `$ref` string value contains a URI [RFC3986](https://tools.ietf.org/html/rfc3986), which identifies the location of the value being referenced.
```

Comment:
We need to decide what to do about our use of the term URI.  If the reference is relatative, then it is technically not a URI.

Reply:
s/URI/uri-reference/

Reply:
Agreed - "URI Reference" as per https://tools.ietf.org/html/rfc3986#section-4.1

Reply:
We're already beyond the point of not changing semantics ;)